assets/tests.ts

```diff
@@ -57,6 +57,7 @@ window.newsroom = {
         display_all_versions_toggle: false,
         display_credits: false,
         display_news_only: true,
+        display_user_role: true,
```

Comment:
we should probably call that `author_role` not `user_role` that seems like it's related to users management